src/main/java/Duke.java

```diff
@@ -1,10 +1,264 @@
+import src.main.java.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.lang.Integer.parseInt;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    int index;
+    List<Task> ls;
+
+    public Duke() {
+        this.index = 0;
+        this.ls = new ArrayList<>();
+    }
+
+    // prints introduction of klaun
```

Comment:
Good to see a comment explaining the method. I think it would be better if they are written in javadoc format like

```
/**
 * Prints introduction of klaun. This method is mainly called when the bot starts up.
 */
```